share/static/html/home.html

```diff
@@ -23,20 +23,20 @@
       {% if user.running %}
       <a id="stop" class="btn btn-lg btn-danger">Stop My Server</a>
       {% endif %}
-      {% if not user.stop_pending %}
+  {% if not user.stop_pending %}
       <a id="start" class="btn btn-lg btn-success"
-	 {% if user.running %}
-	 href="/user/{{user.name}}/"
-	 {% else %}
-	 href="{{base_url}}spawn"
-	 {% endif %}
-	 >
+    	 {% if user.running %}
+    	 href="/user/{{user.name}}/"
+    	 {% else %}
+    	 href="{{base_url}}spawn"
+    	 {% endif %}
+	    >
```

Comment:
It was pretty hard to guess where the block
`{% if not user.stop_pending %}`
ends